libs/accounts/recovery-phone/src/lib/recovery-phone.service.ts

```diff
@@ -239,38 +256,26 @@ export class RecoveryPhoneService {
    *
    * @param phoneNumber The actual phone number
    * @param lastN The last N number of digits to show
-   * @returns A masked number
-   *
-   * @remarks This will not mask a + symbol, since this technically isn't part of the
-   * number. e.g. +15005551234 would be masked as +*******1234 if lastN was 4.
+   * @returns The last N number of digits of the phone number
    */
-  public maskPhoneNumber(phoneNumber: string, lastN?: number) {
+  public stripPhoneNumber(phoneNumber: string, lastN?: number) {
     if (!this.config.enabled) {
       throw new RecoveryPhoneNotEnabled();
     }
-
-    // The + notation can be confusing in a masked number. Don't count it
-    // as a digit.
-    let prefix = '';
-    if (phoneNumber.startsWith('+')) {
-      prefix = '+';
-      phoneNumber = phoneNumber.substring(1);
-    }
-
-    // Clamp lastN between 0 and phoneNumber.length
+    // No stripping needed, session is verified
     if (lastN === undefined) {
-      lastN = phoneNumber.length;
-    } else if (lastN > phoneNumber.length) {
-      lastN = phoneNumber.length;
-    } else if (lastN < 0) {
-      lastN = 0;
+      return phoneNumber;
     }
 
-    // Create mask
-    const maskedPhoneNumber = phoneNumber
-      .substring(phoneNumber.length - lastN)
-      .padStart(phoneNumber.length, '•');
-    return `${prefix}${maskedPhoneNumber}`;
+    const digits = phoneNumber.replace(/\D/g, '');
+
+    // Clamp lastN between 0 and digits.length
+    if (lastN > digits.length) {
+      lastN = digits.length;
+    } else if (lastN <= 0) {
```

Comment:
Semantically this doesn't make sense to me. Would would a negaive lastN digits even mean? I'd error or set to 0 as I did above.

Reply:
I agree - I actually wasn't sure why we allowed a negative number in the other function, but right now this basically treats negative numbers the same as 0. I've changed this locally to just return an empty string for <= 0 as I think that's more clear.